### ISSUE TITLE
align biomaterial type disease properties by updating cell_line.json

### ISSUE THREAD

**Issue by jahilton** [HumanCellAtlas/metadata-schema#1162]
**For which schema is a change/update being suggested?**

cell_line.json

**What should the change/update be?**

`field name` update `disease` to `diseases`
`description` update to `Short description of known disease(s) of the cell line.`
`type` update to `array`
`user_friendly` update to `Known disease(s)`
add `guidelines` `Enter 'normal' if no known disease.`

**Why is the change requested?**

Align this with the field & its description to the `diseases` field in donor_organism.json & specimen_from_organism.json
Enable the capturing of all relevant information if a submitter wishes to submit a cell line sample with multiple known diseases.

**Comment by ESapenaVentura** [comment]
I agree that consistency is key and that we should update this, but I would like to point out that this is a major change and should be discussed with the other components before taking action!

**Comment by zperova** [comment]
@jahilton In the `donor_organism.json` and `specimen_from_organism.json` knowns diseases will be changed to Disease status: https://github.com/HumanCellAtlas/metadata-schema/issues/1124

**Comment by jahilton** [comment]
@zperova, thanks for pointing that out. I updated the ticket description
